2021-04-13 16:27:48,948 INFO    MainThread:32932 [wandb_setup.py:_flush():69] setting env: {}
2021-04-13 16:27:48,949 INFO    MainThread:32932 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-13 16:27:48,949 INFO    MainThread:32932 [wandb_init.py:_log_setup():336] Logging user logs to C:\Users\Jaimie\Documents\VU\Master jaar 1\Data Mining Techniques\DataMiningTechniques\wandb\run-20210413_162748-hmm5xlso\logs\debug.log
2021-04-13 16:27:48,949 INFO    MainThread:32932 [wandb_init.py:_log_setup():337] Logging internal logs to C:\Users\Jaimie\Documents\VU\Master jaar 1\Data Mining Techniques\DataMiningTechniques\wandb\run-20210413_162748-hmm5xlso\logs\debug-internal.log
2021-04-13 16:27:48,949 INFO    MainThread:32932 [wandb_init.py:init():369] calling init triggers
2021-04-13 16:27:48,975 INFO    MainThread:32932 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-13 16:27:48,975 INFO    MainThread:32932 [wandb_init.py:init():418] starting backend
2021-04-13 16:27:48,975 INFO    MainThread:32932 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=spawn, using: spawn
2021-04-13 16:27:48,982 INFO    MainThread:32932 [backend.py:ensure_launched():123] starting backend process...
2021-04-13 16:27:49,543 INFO    MainThread:32932 [backend.py:ensure_launched():128] started backend process with pid: 36456
2021-04-13 16:27:49,544 INFO    MainThread:32932 [wandb_init.py:init():423] backend started and connected
2021-04-13 16:27:49,545 INFO    MainThread:32932 [wandb_init.py:init():463] updated telemetry
2021-04-13 16:27:49,545 INFO    MainThread:32932 [wandb_init.py:init():482] communicating current version
2021-04-13 16:27:50,941 INFO    MainThread:32932 [wandb_init.py:init():487] got version response 
2021-04-13 16:27:50,941 INFO    MainThread:32932 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-13 16:27:51,419 INFO    MainThread:32932 [wandb_init.py:init():520] starting run threads in backend
2021-04-13 16:27:51,980 INFO    MainThread:32932 [wandb_run.py:_console_start():1524] atexit reg
2021-04-13 16:27:51,981 INFO    MainThread:32932 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.WRAP
2021-04-13 16:27:51,982 INFO    MainThread:32932 [wandb_run.py:_redirect():1414] Wrapping output streams.
2021-04-13 16:27:51,982 INFO    MainThread:32932 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-13 16:27:51,982 INFO    MainThread:32932 [wandb_init.py:init():544] run started, returning control to user process
2021-04-13 16:27:52,559 INFO    MainThread:32932 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-13 16:27:52,559 INFO    MainThread:32932 [wandb_run.py:_restore():1466] restore
2021-04-13 16:27:52,797 INFO    MainThread:32932 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 926
  total_bytes: 926
}

2021-04-13 16:27:53,170 INFO    MainThread:32932 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 926
  total_bytes: 926
}

2021-04-13 16:27:53,481 INFO    MainThread:32932 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 926
  total_bytes: 2676
}

2021-04-13 16:27:53,898 INFO    MainThread:32932 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2676
  total_bytes: 2676
}

2021-04-13 16:27:54,013 INFO    MainThread:32932 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2676
  total_bytes: 2676
}

2021-04-13 16:27:54,122 INFO    MainThread:32932 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2676
  total_bytes: 2676
}

2021-04-13 16:27:54,230 INFO    MainThread:32932 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2676
  total_bytes: 2676
}

2021-04-13 16:27:54,341 INFO    MainThread:32932 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2676
  total_bytes: 2676
}

2021-04-13 16:27:55,506 INFO    MainThread:32932 [wandb_run.py:_show_files():1838] logging synced files
